File: JavaScriptBasics/AdvancedConditionalStatements/Exercise/FishingBoat.js
function FishingBoat(input){

    let budget = Number(input[0]);
    let season = input[1];
    let count = Number(input[2]);
    let tax;
    switch(season){
         case "Spring" :
            tax = 3000;break;
            case "Summer" :
                case "Autumn" :
                    tax = 4200;break;
                    case "Winter" :
                        tax = 2600;break;
    }
    if(count<7){
        tax=tax*0.90;
    }else if(count<12 && count>6){
          tax=tax*0.85;
    }else{
        tax=tax*0.75;
    }

    if(count%2==0 && season!="Autumn"){
        tax=tax*0.95;
    }

    if(tax<budget){
        console.log("Yes! You have "+(budget-tax).toFixed(2) +" leva left.");
    }else{
        console.log("Not enough money! You need " + (tax-budget).toFixed(2)+" leva.");
    }

}

File: JavaBasics/AdvancedConditionalStatements/Exercise/FishingBoat.java
package EntryModule.AdvancedConditionalStatements.Exercise;

import java.util.Scanner;
public class FishingBoat {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int budget = Integer.parseInt(scanner.nextLine());
        String season = scanner.nextLine();
        int number = Integer.parseInt(scanner.nextLine());

        int rent = 1;
        double ratio = 1;
        double result;

        if(season.equals("Spring")){rent=3000;}
        if(season.equals("Summer")){rent=4200;}
        if(season.equals("Autumn")){rent=4200;}
        if(season.equals("Winter")){rent=2600;}

        if(number<=6){ratio=0.9;}
        if(number>=7 && 11>=number){ratio=0.85;}
        if(number>=12){ratio=0.75;}

        if(number%2==0 && !season.equals("Autumn")){ratio=ratio*0.95;}

        result=rent*ratio;
        if(result<=budget){
            System.out.printf("Yes! You have %.2f leva left.", budget-result);
        }else{System.out.printf("Not enough money! You need %.2f leva.", result-budget);}


    }
}
